package.json

```diff
@@ -150,7 +150,7 @@
     "access": "public"
   },
   "engines": {
-    "node": ">=12.0.0",
-    "pnpm": ">=5"
+    "node": ">=12.17",
+    "pnpm": ">=6"
```

Comment:
Not sure if engines on pnpm is needed? `"pnpm": ">=6"`

Reply:
I don't think so. Just make sure to have the lock files for npm and yarn in gitignore